packages/mobile/e2e/global-setup.js

```diff
@@ -0,0 +1,42 @@
+const fs = require('fs-extra')
+const { execSync } = require('child_process')
+const detox = require('detox')
+
+async function globalSetup() {
+  const config = resolveSelectedConfiguration() || {}
+  downloadTestButlerAPKIfNeeded(config)
+  await detox.globalInit()
```

Comment:
do we need to handle an error if this promise fails to resolve for whatever reason? same with the `globalCleanup`?

Reply:
I was able to run it 10 times over the course of today and didn't see any failures. If it fails I believe the e2e test suite should fail.